installer/bundle/bundle_skel_Linux.sh

```diff
@@ -584,17 +562,24 @@ case "$installMode" in
         if [ $PROVIDER_ONLY -eq 0 ]; then
```

Comment:
As I recall, a "provider only" build includes just SCX, and not everything else. This was done for a bit for Azure, but I don't think we're doing that anymore. Perhaps this is a good time to rip that out? Or perhaps better to rip that out as a separate piece of work after this. But I don't think we need the notion of provider only builds anymore.